client/my-sites/site-settings/publishing-tools/index.jsx

```diff
@@ -21,7 +21,7 @@ import FormLegend from 'components/forms/form-legend';
 import FormFieldset from 'components/forms/form-fieldset';
 import FormLabel from 'components/forms/form-label';
 import { getSelectedSiteId } from 'state/ui/selectors';
-import { regeneratePostByEmail } from 'state/jetpack/settings/actions';
+import { regeneratePostByEmail } from 'state/jetpack-onboarding/actions';
```

Comment:
❌ Regression here:

1. Go to http://calypso.localhost:3000/settings/writing/:site
2. Verify the "Post by Email" module is enabled in the "Publishing Tools" card and you have an email generated.
3. Click one of the "Save Settings" buttons above, or toggle some of the autosaving settings.
4. The email disappears.

Reply:
Should be fixed by #24082 and rebasing this PR.